doc/monads/states.lean

```diff
@@ -0,0 +1,252 @@
+import Bootstrap.Data.HashMap
```

Comment:
By the way @leodemoura I just want to call out I had to change this import recently from `import Std.Data.HashMap`, so was this breaking change intentional?

Reply:
As I mentioned on Zulip, this is a temporary state of affairs. If you want to keep it compiling, you need to change it to `import Lean.Data.HashMap`, but we will want to guide people to use the Std library, which will locate this at `import Std.Data.HashMap`. (Not sure whether adding Std to the docs will cause boostrapping issues.)

Reply:
![image](https://user-images.githubusercontent.com/18707114/187769077-03a6bca5-643b-41d3-a554-a6dc682c9ca9.png)

Reply:
You need to change it to `import Lean.Data.HashMap` [tomorrow](https://github.com/leanprover/lean4/commit/ebb5b97d73e3ad). Like I said, things are in flux.

Reply:
I see, will do, thanks for the link.

Reply:
Ok, I tested it with my own local build of master and it works great so I checked in the change to `import Lean.Data.HashMap`.